usr/share/rear/conf/default.conf

```diff
@@ -3289,37 +3289,44 @@ SIMPLIFY_TEAMING=no
 
 ####
 # Serial console support for the ReaR rescue/recovery system:
-# By default serial consoles get enabled if serial devices are found and
-# then matching kernel command line parameters like 'console=ttyS0,9600 console=ttyS1,9600'
-# are set when booting the rescue/recovery system (see KERNEL_CMDLINE above).
 # IA64 platforms require it and people need it when no VGA console is available.
+# By default there is serial console support in the ReaR recovery system
+# when 'getty' or 'agetty' and 'stty' can be found,
+# otherwise USE_SERIAL_CONSOLE is automatically set to 'no'.
```

Comment:
What will the kernel command line `console=` parameters be if one sets USE_SERIAL_CONSOLE=no (or if ReaR sets it automatically)?

Reply:
When USE_SERIAL_CONSOLE=no
kernel command line console= parameters on the original system
will not be copied as kernel command line parameters
for the kernel of the ReaR recovery system.
This is needed when the user specifies USE_SERIAL_CONSOLE=no
because then ReaR has to obey.